simpeg_drivers-assets/uijson/direct_current_forward_2d.ui.json

```diff
@@ -35,9 +37,28 @@
         "label": "Elevation channel",
         "tooltip": "Set elevation from channel",
         "parent": "topography_object",
+        "dependency": "topography_object",
+        "dependencyType": "enabled",
```

Comment:
I would think that topography_object should be non-optional and then topography has a disabled dependency on active_model.

Reply:
The active_model becomes non-optional if topo is off, as desired.

![image](https://github.com/user-attachments/assets/83456633-7253-4efd-9835-a5f56cdfa39c)